src/main/java/net/neoforged/moddevgradle/internal/IdeIntegration.java

```diff
@@ -103,4 +103,13 @@ void configureTesting(SetProperty<ModModel> loadedMods,
                           Provider<RegularFile> vmArgsFile) {
     }
 
+    protected boolean shouldGenerateRunFor(RunModel run) {
+        // The user can disable the IDE run directly...
+        final var shouldGenerate = run.getIdeConfigGenerated().get();
```

Comment:
First appearance of `final var` in MDG... Why?

Reply:
My university has drilled it into our minds that we should make things as `final` as possible, if it's against the style of MDG I can remove it, since it's effectively final anyway.

Reply:
I think the noise outweighs the benefits for local variables and parameters. So yeah I'd remove it. (And the one below too).